packages/@okta/vuepress-site/docs/reference/hook-best-practices/index.md

```diff
@@ -0,0 +1,82 @@
+* Process the Hook request data after the response is sent to avoid delays or timeouts.
+
+A timeout of three seconds is enforced on all outbound requests for Event and Inline Hooks, with one retry in the event of a timeout or an error response from the external service. If a successful response has not been received after that, a 400 error is returned with more information about what failed.
+
+See Inline Hook [Timeout and Retry](/docs/concepts/inline-hooks/#timeout-and-retry) for further information on Inline Hook timeout and retry behavior; see Event Hook [Timeout and Retry](https://developer.okta.com/docs/concepts/event-hooks/#timeout-and-retry) for the same.
+
+## Limits, duplicates, and order of Hook calls
+
+The number of Hook calls and the limits per org are available in the following table. Keep in mind these numbers and limits when designing your Hook solution.
+
+The order of Event or Inline Hook calls is not guaranteed. Your external service processing of the Hook requests must consider this for your solution. Also, to avoid processing duplicate requests, use the `eventId` property to identify unique requests.
+
+| Hook Type | Limit Type | Limit | Description |
+| --------- | -----------| ----- | ----------- |
+| Event Hook | Number of daily Event Hooks | 100K | A maximum of 100 thousand Event Hooks can be fired, per org, per day. Event Hooks are not recorded or replayed after this point. If a request times out after three seconds, Event Hooks are retried one time afterward. Retries do not count toward the org limit.
+|            | Maximum number of Event Hooks per org | 10 | A maximum of 10 active Event Hooks can be configured per org. Each Event Hook can be configured to deliver multiple event types. |
+| Inline Hook | Timeout | 3 seconds | Okta Inline Hooks have a completion timeout of three seconds with a single retry. However, a request is not retried if your endpoint returns a 4xx HTTP error code. Any 2xx code is considered successful, and the request is not retried. If the external service endpoint responds with a redirect, it is not followed. |
+|             | Maximum number of Inline Hooks per org | 50 | The maximum number of Inline Hooks that can be configured per org is 50, which is a combined total for any combination of Inline Hook types. |
+
+## Troubleshoot your Hook implementations
+
+Developers and administrators can preview sample Okta calls and responses from your external service for all Event Hooks ([Event Hook preview](https://help.okta.com/en/prod/Content/Topics/automation-hooks/event-hooks-preview.htm)) and for SAML and Registration Inline Hooks ([Inline Hook preview](https://help.okta.com/en/prod/Content/Topics/automation-hooks/preview-inline-hooks.htm)). The preview pages provide sample request and response code prior to enabling the Hooks to assist in your set up and external service development.
```

Comment:
~The preview pages provide sample request and response code prior to enabling the Hooks to assist in your set up and external service development.~
(you already said this in the previous sentence)

Reply:
I guess I wanted to highlight the UI Preview Page, but that's fairly intuitive and I'm linking to help topics anyway. Deleted.